### PR TITLE
fix: Modal title removes leading/trailing parens

### PR DESCRIPTION
**Pull request by jbrundage**
## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [x] The change has been fully tested:
  - [ ] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [x] I have addressed them here
  - [ ] I have raised new issues to address them separately

## Testing:
Verified this change within DSP-DEV

### CODE REVIEW

File: src/layout/Modal.js [hunk 1]
```diff
@@ -37,6 +37,8 @@
     Modal.prototype.publish("maxHeight", "800px", "string", "maxHeight");
     Modal.prototype.publish("fixedWidth", null, "string", "fixedWidth");
     Modal.prototype.publish("fixedHeight", null, "string", "fixedHeight");
+    Modal.prototype.publish("overflowX", "hidden", "string", "overflowX");
+    Modal.prototype.publish("overflowY", "scroll", "string", "overflowY");
```
Comment (GordonSmith) [inline src/layout/Modal.js]: Are these changes related to this PR?

File: src/layout/Modal.js [hunk 2]
```diff
@@ -130,9 +132,11 @@
         this._modalBody = this._modal.append('div')
             .classed('layout_Modal-body',true)
             .style('padding',this.bodyPadding())
+            .style('overflow-x', this.overflowX())
+            .style('overflow-y', this.overflowY())
```
Comment (GordonSmith) [inline src/layout/Modal.js]: Are these changes related to this PR?

File: src/layout/Modal.js (hunk 1 above)
Comment (jbrundage) [reply src/layout/Modal.js]: Yes, it fixes #3245

File: src/layout/Modal.js (hunk 2 above)
Comment (jbrundage) [reply src/layout/Modal.js]: Yes, it fixes #3245
